web-ng/src/app/services/notification/notification.service.spec.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { NotificationService } from './notification.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+describe('NotificationService', () => {
+  let matSnackBar: jasmine.SpyObj<MatSnackBar>;
+  let service: NotificationService;
+
+  beforeEach(() => {
+    matSnackBar = jasmine.createSpyObj('MatSnackBar', ['open']);
+    TestBed.configureTestingModule({
+      providers: [{ provide: MatSnackBar, useValue: matSnackBar }],
+    });
+    service = TestBed.inject(NotificationService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('success', () => {
```

Comment:
This is probably a stupid q, but what's the advantage of having this sub-describes over just having multiple it()'s?

Reply:
For methods with multiple cases it groups tests together in code and in outputs without us needed to repeat the method name in each test description.